applications/tari_base_node/src/builder.rs

```diff
@@ -425,7 +425,7 @@ where
     let mut wallet_db_folder = PathBuf::from(wallet_file);
     wallet_db_folder.set_extension("dat");
     let wallet_path = PathBuf::from(wallet_db_folder.file_stem().expect("unable to get wallet db path"));
-    let _ = std::fs::create_dir_all(&wallet_path).unwrap_or_default();
+    std::fs::create_dir_all(&wallet_path).unwrap_or_default();
```

Comment:
would this not through a unused result?

Reply:
Function signature is this:
```
pub fn create_dir_all<P: AsRef<Path>>(path: P) -> Result<()>
```

Suggestions?